### PR TITLE
Add more OTP versions to GH Actions

### PR DESCRIPTION
**Pull request by vkatsuba**
**Purposes of changes:**
* Added more OTP versions `21.3`, `22.3`, `23.0.4`, `23.2.7.0`|
* Update section `on` for run `GH Actions` in develop branches

### CODE REVIEW

File: .github/workflows/erlang.yml [hunk 1]
```diff
@@ -2,18 +2,23 @@ name: Erlang CI
...
 
 jobs:
 
   build:
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        otp: [21.3, 22.0.7, 22.3, 23.0.4, 23.2.7.0]
```
Comment (elbrujohalcon) [inline .github/workflows/erlang.yml:19]: This is fine for now, but we don't officially support anything below `22`. If, at some point, tests start failing on `21.3`, we'll not fix the system, we will just remove `21.3` from here.

**Review by elbrujohalcon — APPROVED** [review]

File: .github/workflows/erlang.yml (hunk 1 above)
Comment (vkatsuba) [reply .github/workflows/erlang.yml:19]: Maybe make sense already remove `21.3` and provide some info about minimum support version in `README.md` based on `matrix -> otp` section in `erlang.yml`?

**Comment by elbrujohalcon** [comment]
Humm… not sure why `build` is not running for this PR with the changes… Do you know what's going on, @vkatsuba ?

**Comment by vkatsuba** [comment]
> Humm… not sure why `build` is not running for this PR with the changes… Do you know what's going on, @vkatsuba ?

Yep, cuz I provide PR from fork.

**Comment by elbrujohalcon** [comment]
Alright… I'll merge it then 🤞

**Pull request merged**